tests/models/test_dagbag.py

```diff
@@ -56,15 +55,17 @@ def setup_class(cls):
     def teardown_class(cls):
         shutil.rmtree(cls.empty_dir)
 
-    def setup_methods(self) -> None:
+    def setup_method(self) -> None:
         db.clear_db_dags()
         db.clear_db_runs()
         db.clear_db_serialized_dags()
+        db.clear_dag_specific_permissions()
```

Comment:
Clearing things before every test is too exccessive. It’s probably better to change this to `setup_class` instead.

Reply:
Fixed